files/qsub.pl

```diff
@@ -540,7 +540,19 @@ sub parse_script
     # replace PBS_JOBID in -o / -e
     #   torque sets stdout/err in cwd -> so force abspath like done above
     # All script changes here
+    my $change_line = sub {
```

Comment:
I would give this a more meaningful name, like `replace_env_vars`?

Also, please include a line or two of comments above to describe what it does, give an example?